dask_expr/expr.py

```diff
@@ -703,8 +696,24 @@ def find_operations(self, operation: type) -> Generator[Expr]:
             for dep in node.dependencies():
                 stack.append(dep)
 
-            if isinstance(node, operation):
-                yield node
+            yield node
+
+    def find_operations(self, operation: type) -> Generator[Expr]:
```

Comment:
I think that using multiple operation types here would be helpful

Reply:
Sure.  Pushed.